Missing dependency babel-eslint

I get the following error after `clone` and then `npm i 

````
npm i

> rollup-starter-project@1.0.0 prepublish C:\Users\dotnet\Projects\playground\rollup-starter-project
> npm run build && npm test


> rollup-starter-project@1.0.0 prebuild C:\Users\dotnet\Projects\playground\rollup-starter-project
> eslint lib test


C:\Users\dotnet\Projects\playground\rollup-starter-project\lib\index.js
  0:0  error  Cannot find module 'babel-eslint'
```

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
   ],
   "homepage": "https://github.com/eventualbuddha/rollup-starter-project#readme",
   "devDependencies": {
+    "babel-eslint": "^5.0.0-beta4",
     "babel-preset-es2015-rollup": "^1.0.0",
     "eslint": "^1.2.0",
     "mocha": "^2.2.5",

